digital-workspace/4.3/develop/manage.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Manage Digital Workspace
+---
+
+Learn how to run the Digital Workspace in your local development environment from the source code and manage it from a developer perspective.
+
+## Prerequisites and requirements
+
+* Alfresco Content Services - Enterprise Edition:
+  * Open your browser and check everything starts up correctly:
+
+    ```bash
+    http://localhost:8080/alfresco
+    ```
+
+* `Node.js` 18.x
+
+## Clone and launch the front-end application
+
+Once Content Services is up and running, you must make the source code of the project available locally in your development environment. If you're an Alfresco customer or partner, you can get a local copy of the project by opening a request in [Hyland Community](https://community.hyland.com/tskb){:target="_blank"}.
+
+1. In the `alfresco-apps` directory, create a file named `.env` with the following content.
+
+    Enter the Content Services URL as the `APP_CONFIG_ECM_HOST` value.
+
+    ```bash
+    ACA_BRANCH="2.10.0"
```

Comment:
I believe that's no longer used

Reply:
You can remove `ACA_BRANCH="2.10.0"` line, it doesn't have any replacement it's just obsolete